src/scheduler/AsyncTask.php

```diff
@@ -230,7 +231,7 @@ protected function storeLocal(string $key, mixed $complexData) : void{
 	 */
 	protected function fetchLocal(string $key){
 		$id = spl_object_id($this);
-		if(!isset(self::$threadLocalStorage[$id][$key])){
+		if(!isset(self::$threadLocalStorage[$id]) || !array_key_exists($key, self::$threadLocalStorage[$id])){
```

Comment:
Couldn't this just use `array_key_exists()`? `isset()` is now redundant.

Reply:
If I remove the `isset()` check, when you try to call `fetchLocal()` while never having done `storeLocal()` it will cause an "Undefined array key" error instead of throwing the proper exception.

Reply:
Oh, I get it now